### ISSUE TITLE
Feature request: Create module for Domain WHOIS metrics

### ISSUE THREAD

**Issue by nonamef** [prometheus/blackbox_exporter#759]
Would be good to have a Blackbox module to export Domain WHOIS metrics. 
**Example project**: https://github.com/caarlos0/domain_exporter

**Comment by SuperQ** [comment]
This seems out of scope for the blackbox_exporter. Especially given there's already a dedicated exporter for this.

**Comment by roidelapluie** [comment]
Yes, however I can see how we could get the TTL out of the dns probes we currently do.

**Comment by electron0zero** [comment]
closing because this request is out of scope for the project, we recommend users to use another exporter like https://github.com/caarlos0/domain_exporter for getting WHOIS metrics.
